src/index.html

```diff
@@ -22,6 +22,359 @@
+              </li>
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Samsung
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Smartphones
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                  data-qa="nav-hover"
+                >
+                  Laptops & Computers
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Gadgets
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Tablets
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Photo
+                </a>
+              </li>
+
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                >
+                  Video
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </header>
+
+    <main class="main">
+      <section class="catalog">
+        <div class="catalog__content">
+          <div
+            class="card"
+            data-qa="card"
+          >
+            <img
+              src="./images/imac.jpeg"
+              alt="card image"
+              class="card__image"
+            />
+            <h2 class="card__title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h2>
+            <p class="card__code">Product code: 195434</p>
+            <div class="card__rating rating">
+              <div class="rating__stars stars stars--4">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+              <p class="rating__reviews">Reviews: 5</p>
+            </div>
+            <p class="card__price price">
+              <span class="price__text">Price:</span>
+              <span class="price__amount">$2,199</span>
+            </p>
+
+            <a
+              href="#"
+              class="card__buy"
+              data-qa="card-hover"
+            >
+              BUY
+            </a>
+          </div>
+          <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
          </div>
          
          <div class="card">
```